docs/griptape-framework/drivers/file-manager-drivers.md

```diff
@@ -0,0 +1,48 @@
+---
+search:
+  boost: 2
+---
+
+## Overview
+
+File Manager Drivers can be used to load and save files with local or external file systems.
+
+You can use File Manager Drivers with Loaders:
+
+```python
+--8<-- "docs/griptape-framework/drivers/src/file_manager_driver.py"
+```
+
+Or use them independently as shown below for each driver:
+
+## File Manager Drivers
+
+### Griptape Cloud
+
+!!! info
+    This driver requires the `drivers-file-manager-griptape-cloud` [extra](../index.md#extras).
+
+The [GriptapeCloudFileManagerDriver](../../reference/griptape/drivers/file_manager/griptape_cloud_file_manager_driver.md) allows you to load and save files sourced from Griptape Cloud Asset and Bucket resources.
+
+```python
+--8<-- "docs/griptape-framework/drivers/src/griptape_cloud_file_manager_driver.py"
+```
+
+### Local
+
+The [LocalFileManagerDriver](../../reference/griptape/drivers/file_manager/local_file_manager_driver.md) allows you to load and save files sourced from a local directory.
+
+```python
+--8<-- "docs/griptape-framework/drivers/src/local_file_manager_driver.py"
+```
+
+### Amazon S3
+
+!!! info
+    This driver requires the `drivers-web-search-amazon-s3` [extra](../index.md#extras), and a Tavily [api key](https://app.tavily.com).
```

Comment:
Wrong extra, and it doesn't require tavily

Reply:
I suck